### ISSUE TITLE
could you write some detile document?

### ISSUE THREAD

**Issue by nolotus** [Stefdv/Gun-Server-CMS-App#1]
suah as deploy 
plugin 
or  some notice thing

**Comment by Stefdv** [comment]
yes, i can but.can you be more specific in what you're looking for? Are you interested in the Gun part or the nw app itself?

**Comment by nolotus** [comment]
yes  gun ,best all,
maybe both 
ahaha ~~
